CONTRIBUTING.md

```diff
@@ -1,52 +1,117 @@
+that you should take a look at.
+
+## Connecting with the Community
+
+You might already know exactly what you want to contribute and how.
+If not, if could be useful to interact with existing developers.
+There are several easy ways to do this.
+
+### Slack Channel
+
+The best place to interact with the CoCo community is the
+`#confidential-containers` channel in the [CNCF Slack workspace](https://slack.cncf.io/).
+This is a great place to ask any questions about the project
+and to float ideas for future development.
+
+### Community Meeting
+
+CoCo also has a weekly community meeting on Thursdays.
+See [the agenda](https://docs.google.com/document/d/1E3GLCzNgrcigUlgWAZYlgqNTdVwiMwCRTJ0QnJhLZGA/)
+for more information.
+The community meeting usually has a packed agenda, but there is always time
+for a few basic questions.
+
+### GitHub Issue
+
+You can also open issues in GitHub.
+Try to open your issue on the repository that is most closely related
+to your question.
+If you aren't sure which repository is most relevant, you can open an issue
+on this repository.
+If you're creating an issue that you plan to resolve, consider noting this in
+a comment so other developers know that someone is working on the problem.
+
+## Making a Contribution
+
+The mechanics of making a contribution are relatively straightforward.
+All contributions should be made as GitHub pull requests.
+Make sure your PR includes a Developer Certificate of Origin
+and follow any existing stylistic or organizational conventions.
+These requirements are explained in more detail below.
+
+Some contributions are simpler than others.
+If you are new to the community it could be good to start with
+a few simple contributions to get used to the process.
+Larger contributions, especially ones that span multiple subprojects,
+or have implications on the trust model or project architecture,
+will usually require more discussion and review.
+Nonetheless, CoCo has a track record of [responding to PRs quickly](https://confidentialcontainers.devstats.cncf.io/d/10/pr-time-to-engagement)
+and [merging PRs quickly](https://confidentialcontainers.devstats.cncf.io/d/16/opened-to-merged).
+
+If you are preparing a large contribution, it is wise to share your idea
+with the community as early as possible. Consider making an `RFC` issue
+that explains the changes. You might also try to break large contributions
+into smaller steps.
+
+### Making a Pull Request
+
+If you aren't familiar with Git or the GitHub PR workflow, take a look at [this section](https://github.com/kata-containers/community/blob/main/CONTRIBUTING.md#github-workflow)
+of the Kata Containers contributor guide.
+We have a few firm formatting requirements that you must follow,
+but in general if you are thoughtful about your work and responsive to review,
+you shouldn't have any issues getting your PRs merged.
+The requirements that we do have are there to make everyone's life a little easier.
```

Comment:
So Kata has requirements like the backport labels and references to tooling I don't think we've imported to coco. Would it be better to keep the github-workflow section in this document and cut out the kata-specific stuff?

Reply:
Let me change the wording a bit. Contributing to Kata is still a way to contribute to CoCo, so I think we should still point people to the Kata guide.